website/blog/2025-11-24-aks-automatic-managed-system-node-pools/index.md

```diff
@@ -0,0 +1,172 @@
+
+Since AKS manages the system node pool on your behalf, additional restrictions protect platform stability. User workloads cannot run on the managed system node pool and all create, update, and delete operations on managed system pool resources are denied, as are pod `exec` and `attach` operations.
+
+**Preventing container escapes:** Blocking privileged containers, host namespaces, host ports, and hostPath volumes keeps security incidents contained to a single workload rather than spreading across the cluster.
+
+**Reducing attack surface:** Restricting Linux capabilities to a minimal set means processes run with only the permissions they need. Fewer capabilities translate directly to fewer exploitation opportunities.
+
+**Enforcing least privilege:** Requiring containers to run as non-root and disabling privilege escalation limits the blast radius of any vulnerability.
+
+**Maintaining kernel protections:** Seccomp, AppArmor, and SELinux profiles filter system calls and confine container behavior. Policies ensure these protections stay active.
+
+**Enabling safe cluster operations:** Limiting sysctls to safe parameters and protecting node objects ensures platform components run undisturbed and node drains proceed smoothly.
+
+For detailed specifications, see the [Deployment Safeguards documentation](https://learn.microsoft.com/azure/aks/deployment-safeguards).
+
+## Pod Readiness SLA for AKS Automatic
+
+![API Server Uptime SLA: 99.95% control plane uptime. Pod Readiness SLA: 99.9% operations under 5 minutes](automatic-slas.png)
+
+Uptime means more than a healthy control plane; it means your applications are actually serving users. The [Pod Readiness SLA](https://www.microsoft.com/licensing/docs/view/Service-Level-Agreements-SLA-for-Online-Services) guarantees that pods reach readiness targets, closing the gap between "the cluster is healthy" and "my app is ready."
+
+- **Faster recovery during failures:** Node failures and scale events trigger remediation so pods return to a ready state within defined thresholds.
+- **Predictable reliability:** Availability planning aligns with measurable guarantees instead of best-effort behavior.
+- **Reduced operational overhead:** Platform automation handles remediation, eliminating manual firefighting during disruptions.
+- **Business continuity at scale:** Mission-critical services experience minimal disruption even during infrastructure events.
+
+## Pricing
+
+AKS Automatic pricing includes a fixed monthly cluster fee and per-vCPU charges on top of standard VM compute costs. This pricing includes financially backed SLAs for both API server uptime and pod readiness. For current rates and a full breakdown by VM category, see the [Azure Kubernetes Service pricing page](https://azure.microsoft.com/pricing/details/kubernetes-service#pricing).
+
+## Getting started

```

Comment:
Can we leave the How to and CLI parts to a link of the actual doc instead ? . The behind the scenes thinking, benefits and why is more valuable in the blog IMHO.

Reply:
Let's keep it as discussed.